dspace-api/pom.xml

```diff
@@ -895,6 +913,11 @@
             <artifactId>swagger-core</artifactId>
             <version>1.6.2</version>
           </dependency>
+          <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>2.13.2</version>
```

Comment:
Should this have a `<scope>test</scope>` added?  It appears, to me, that we are not using this library in our code, but maybe it's required by the new `s3mock_2.13` library?  If so, we should make sure this is also limited to `test` scope.